### PR TITLE
Table: Fix table data links so they refer to correct row after sorting

### LINKED ISSUE #32542: Table: Data links incorrect after sorting table column
**What happened**:

table_invalid_link

Url links are not properly generated for table entries.

**What you expected to happen**:

Generating proper URL

**How to reproduce it (as minimally and precisely as possible)**:

1. Add TestData DB.
2. Create Panel Table.
3. Select TestData DB datasource and add couple points in Manual Entry Scenario.
4. Add Override for :
   * Fields with name: value
   * Data links: URL: ${__value.text}
5. Sort table asc/desc by value and check generated URL.

**Anything else we need to know?**:

Version 7.5.1 - everything ok.

**Environment**:
- Grafana version: 7.5.2
- Data source type & version: TestData DB
- OS Grafana is installed on: Debian 9
- User OS & Browser: Ubuntu 20.04, Mozilla Firefox 87.0
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by torkelo**
Fixes #32542

Introduced by
https://github.com/grafana/grafana/pull/32370

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
Great catch, can we test this somehow with RTL?

**Comment by torkelo** [comment]
> Great catch, can we test this somehow with RTL?

Was thinking the same, maybe @kaydelaney is that something you can help me with in this PR?

**Review by thisisobate — APPROVED** [review]
LGTM!

**Comment by torkelo** [comment]
@kaydelaney can you take a look at this, see if there is a way to add a unit test for the sorting

**Comment by hugohaggmark** [comment]
I'm going to push this PR into a mergeable state today

File: packages/grafana-ui/src/components/Table/Table.tsx [hunk 1]
```diff
@@ -27,7 +27,6 @@ import { Icon } from '../Icon/Icon';
 import { CustomScrollbar } from '../CustomScrollbar/CustomScrollbar';
 import { Filter } from './Filter';
 import { TableCell } from './TableCell';
-import ClipboardButtonStoryInternal from '../ClipboardButton/ClipboardButton.story.internal';
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Table/Table.tsx]: Not sure how this got inserted here but should be safe to remove right @torkelo?

**Pull request merged**